src/services/CommandExecutionService.js

```diff
@@ -14,12 +14,12 @@ export default class CommandExecutionService {
   }
 
   executePriceEstimates(startAddress, endAddress, distanceUnitName) {
-    if (typeof startAddress !== 'string') {
-      throw new TypeError('start address should be a string');
-    }
-
-    if (typeof endAddress !== 'string') {
-      throw new TypeError('end address should be a string');
+    // commander.js doesn't support required arguments, and will always
+    // interpolate arguments into strings.
+    if (typeof startAddress !== 'string' || typeof endAddress !== 'string') {
```

Comment:
Do you think that something like
```javascript
if (typeof startAddress !== 'string' || !startAddress.trim().length) {
    throw new TypeError('A start address is required');
}
```
might be clearer (obviously, the same would be done for `endAddress`)?

One advantage is that it separates the validation of `startAddress` and `endAddress`.